src/main/kotlin/org/wycliffeassociates/otter/common/data/model/ContentLabelEnum.kt

```diff
@@ -0,0 +1,19 @@
+package org.wycliffeassociates.otter.common.data.model
+
+import java.util.*
+
+enum class ContentLabelEnum(val value: String, val type: ContentType) {
```

Comment:
The "Enum" class name suffix is just an old habit.  It doesn't follow this project's style, so I'll change it.

Reply:
Fixed.